src/h5web/vis-packs/core/utils.ts

```diff
@@ -21,6 +21,8 @@ const TICK_FORMAT = format('0');
 
 export const DEFAULT_DOMAIN: Domain = [0.1, 1];
 
+const MIN_POSITIVE_VALUE = 1e-250;
```

Comment:
I can't use `Number.MIN_VALUE` as it is already too small.

For a log scale and numbers close to `Number.MIN_VALUE`, `scale.invert` indeed gives `0`. The `isSupported` check then passes giving an extended log domain of `[0, x]` and `NaN`s on the next render.

Reply:
Could you use the `scale` the default way to get the min number supported by `scale.invert`: `scale(Number.MIN_VALUE)`?

Reply:
I feel like the logic should be the same for min and max . Could you pass the scale to `isSupported` to make sure that `scale(Number.MIN_VALUE) < Math.abs(val) < scale(Number.MAX_VALUE)`?